specification/http/src/main/scripts/org/kaazing/specification/http/rfc7231/method.definitions/post/response.rpt

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright 2007-2015, Kaazing Corporation. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+accept http://localhost:8000/index.html
+accepted
+connected
+
+read method "POST"
+read header "Content-Length" "7"
+read "content"
+read closed
+	
+write status "200" "OK"
+write header content-length
```

Comment:
I don't think you need the `write header content-length` for most of these response scripts.  They don't hurt but they don't really add anything IMO